README.md

```diff
@@ -334,7 +334,7 @@ Internal Revenue Service; Bloomberg BNA; American Petroleum Institute; Alcohol a
 
 ### <a id="table12"></a>Table 12. State Individual Income Tax Rates
```

Comment:
Nope, I just messed up the table numbers. My bad. The table numbers did change though and have been edited in the README file.